### PR TITLE
Feat/responsive sustainable

### PR DESCRIPTION
**Pull request by rileydanejohnston**
features:
- refactored sustainable section so there are less components
- put 'benefits' section into it's own component
- responsive styles

I can't think of anything out of the ordinary with this section. I basically refactored many different components into the same component, then used props for the differences like positioning. Let me know if you have any questions!

### CODE REVIEW

File: src/components/benefits/styledBenefits.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  height: 751px;       /* 871px (slide to next section) - 120px (padding) */
+  margin: 0 auto;
+  position: relative;
+
+  @media( max-width: 1280px) {
+  }
+  @media( max-width: 1023px) {
+    height: 878px;
+  }
+  @media(max-width: 767px) {
```
Comment (trisboyd) [inline src/components/benefits/styledBenefits.js:13]: The space on the bottom of this section at 767px is enormous.  It's in line with the design, but I think it's FAR too big.

File: src/components/carousel/Carousel.js [hunk 2]
```diff
@@ -23,28 +28,46 @@ const Carousel = () => {
   }
 
   const scrollPrev = useCallback(() => {
```
Comment (trisboyd) [inline src/components/carousel/Carousel.js:30]: Was that in the documentation for this scroller?  I've heard of useCallback but I have yet to implement it myself.  Nice

File: src/components/benefits/styledBenefits.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    flex-direction: column;
+    justify-contents: center;
+    align-items: center;
+    gap: 75px;
+  }
+`;
+
+export const BenefitGroup = styled.div`
+  display: flex;
+  gap: 40px;
```
Comment (trisboyd) [inline src/components/benefits/styledBenefits.js:26]: The first image/text group has a smaller gap (25px) at higher screen widths, but I wouldn't mind if you didn't change this.  Seems like poor design to make them all different

**Review by trisboyd — APPROVED** [review]
Good call on simplifying the components and separating the benefits section.  I think it's worth it to have the benefits groupings slightly off from the design in order to create a consistent approach to each.  If you want to make small changes though, be my guest

File: src/components/carousel/Carousel.js (hunk 2 above)
Comment (rileydanejohnston) [reply src/components/carousel/Carousel.js:30]: yeah man, it's straight from the docs. it makes me want to check out the other hooks available. There might be some cool stuff there.

File: src/components/benefits/styledBenefits.js (hunk 3 above)
Comment (rileydanejohnston) [reply src/components/benefits/styledBenefits.js:26]: Yeah I've started to get to the point where if there's a small difference like this, I may deviate from the design a bit for convenience

File: src/components/benefits/styledBenefits.js (hunk 1 above)
Comment (rileydanejohnston) [reply src/components/benefits/styledBenefits.js:13]: Yeah there's been some interesting design choices. I'm just gonna roll with it.

**Pull request merged**